packages/nimble-components/src/mapping/mention-user/index.ts

```diff
@@ -0,0 +1,25 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { Mapping } from '../base';
+import type { MentionHref } from '../base/types';
+import { template } from '../base/template';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-mapping-mention-user': MappingMentionUser;
+    }
+}
+/**
+ * Maps key (url) to user display name
+ * Non-visible configuration element for mapping rich text user mention keys to the user name to display
```

Comment:
User mappings could contain other info in the future
```suggestion
 * Maps key (url) to a user
 * Non-visible configuration element for mapping rich text user mention keys to a user
```

Reply:
Updated